tests/pulsar-client-admin-shade-test/src/test/java/org/apache/pulsar/tests/integration/SmokeTest.java

```diff
@@ -87,6 +87,7 @@ public void checkAdmin() throws PulsarClientException, PulsarAdminException {
         expectedNamespacesList.add("public/default");
         expectedNamespacesList.add("public/functions");
         Assert.assertEquals(admin.namespaces().getNamespaces("public"), expectedNamespacesList);
+        admin.brokerStats().getLoadReport();
```

Comment:
would it be possible to also validate that there's valid content?

Reply:
I have added not null validation.

Reply:
Because the standalone uses `org.apache.pulsar.broker. load-balance.NoopLoadManager`, which returns null, we just check that no exception is thrown.